src/ExpressionEvaluator/CSharp/Test/ResultProvider/TupleTests.cs

```diff
@@ -299,102 +299,6 @@ public void LongTuple_NoNames()
             }
         }
 
-        /// <summary>
-        /// If tuple fields are missing, fall back to the default
-        /// display for Value (that is, display the type), and
-        /// drop missing fields from the expansion.
-        /// </summary>
-        [Fact]
-        public void MissingFields()
```

Comment:
>MissingFields [](start = 20, length = 13)

FYI, @cston I removed this test after your review. There is no way to get this test to pass (we can't compile and emit such a bad ValueTuple type).

```
    (15,16): error CS0171: Field '(T1, T2, T3).Item3' must be fully assigned before control is returned to the caller
    (15,16): error CS0171: Field '(T1, T2, T3).Item1' must be fully assigned before control is returned to the caller
    (15,16): error CS0171: Field '(T1, T2, T3).Item2' must be fully assigned before control is returned to the caller
```